bom/pom.xml

```diff
@@ -39,6 +39,7 @@
         <msgpack.version>0.6.12</msgpack.version>
         <protostuff.version>1.5.9</protostuff.version>
         <grpc.version>1.33.1</grpc.version>
+        <fury.version>0.1.1</fury.version>
```

Comment:
Fury ` 0.2.0` has been released, it's much faster and smaller than `0.1.1`, please upgrade to ` 0.2.0`

Reply:
OK，i try to make it